### ISSUE TITLE
Link to `setup_episode` in  the `create` episode is broken. 

### ISSUE THREAD

**Issue by hmacdope** [swcarpentry/git-novice#881]
The link to `02-setup.md` in the line:

```markdown
See the [setup episode](02-setup.md) for more information on this change.
```

in file `03-create.md` does not seem to render correctly.  :)

**Comment by hmacdope** [comment]
I will PR.

**Comment by joenio** [comment]
3 PRs were submited solving this same issue #867 882 #884 

someone can review it?

**Comment by dsmits** [comment]
I will look into those PRs

**Comment by dsmits** [comment]
I just approved #867 . I will leave a comment at the other PRs

**Comment by kekoziar** [comment]
merged #867
